microscope/win32.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python
+"""Win32 specific microscope classes.
+"""
+import os
+import sys
+
+try:
+    import win32api
+except:
+    # Add executable path to DLL search PATH.
+    os.environ['PATH'] += ';' + os.path.split(sys.executable)[0]
+    import win32api
```

Comment:
Is it possible to limit the change of PATH to the function that needs it? This changes it to the whole process (and II'l guess subprocesses).

Reply:
Leaving this as is for now, because its the only way to ensure that the Windows service can start correctly.